plans/vnets.tf

```diff
@@ -42,17 +42,28 @@ resource "azurerm_resource_group" "development" {
 
 ## VIRTUAL NETWORKS
 ################################################################################
+
+resource "azurerm_virtual_network" "public_prod" {
+  name                = "${var.prefix}-jenkins-public-prod"
+  resource_group_name = "${azurerm_resource_group.public_prod.name}"
+  address_space       = ["10.0.0.0/16"]
+  location            = "${var.location}"
+  #depends_on          = ["azurerm_network_interface.ci_trusted_agent_2_nic"]
```

Comment:
should this commented out line just be removed?

Reply:
You are right this must be removed